aiida/cmdline/utils/query/calculation.py

```diff
@@ -80,7 +80,7 @@ def get_filters(self,
             filters[process_state_attribute] = {'in': process_state}
 
         if process_label is not None:
-            filters[process_label_attribute] = process_label
+            filters[process_label_attribute] = {'like': process_label}
```

Comment:
I think the `like` operation is significantly slower than equals. Let's check if the `process_label` contains the wildcard operator in which case we use `like` otherwise use equality operator

Reply:
I see your point. For ~1000 entries in a database of 15K nodes, I did not see much difference. I can add a check to only use `like` when wildcard characters are present in the passed option.